reports/tables.Rmd

```diff
@@ -0,0 +1,250 @@
+---
+title: YouthREX tables
```

Comment:
Ah I get it now. This is the script I should be looking at right? I'm running it now - sorry for the confusion